superset/config.py

```diff
@@ -264,6 +264,12 @@ def _try_json_readsha(filepath: str, length: int) -> str | None:
 # Configuration for scheduling queries from SQL Lab.
 SCHEDULED_QUERIES: dict[str, Any] = {}
 
+# Rate limiting
+RATELIMIT_ENABLED = True
+AUTH_RATE_LIMITED = True
+AUTH_RATE_LIMIT = "2 per 5 second"
+RATELIMIT_APPLICATION = "50 per second"
```

Comment:
@dpgaspar I know these were introduced in FAB 4.3.0. Do you have an idea of what sensible defaults would be for Superset?

Reply:
Airflow implemented the same, their using `auth_rate_limit = 5 per 40 second`. Not sure about rate limiting the entire application this way, rate limits are better when tied to a user and state is set globally, this is possible with [flask-limiter](https://flask-limiter.readthedocs.io/en/stable/) (what FAB is using for this) but needs further configuration.

Reply:
Wait, I though these configuration keys would enable `flask-limiter` in FAB? That's not the case? How do we do that then?

Reply:
it's done exactly like that